builder/image/slugbuilder/builder/build.sh

```diff
@@ -76,7 +76,15 @@ if [[ -n "$BUILDPACK_URL" ]]; then
 
     buildpack="$buildpack_root/custom"
     rm -fr "$buildpack"
-    git clone --quiet --depth=1 "$BUILDPACK_URL" "$buildpack"
+
+    url=${BUILDPACK_URL%#*}
+    branch=${BUILDPACK_URL#*#}
+    if [[ "$branch" == "$url" ]] || [[ -z "${branch}" ]]; then
+        branch="master"
+    fi
+
+    git clone --quiet --branch "$branch" --depth=1 "$url" "$buildpack"
```

Comment:
That function would have to be refactored out into a separate script so it can be used as a shared library